biblemulticonverter/src/main/java/biblemulticonverter/format/paratext/model/Version.java

```diff
@@ -0,0 +1,32 @@
+package biblemulticonverter.format.paratext.model;
+
+public enum Version {
+	V1(1, 0, 0),
+	V2(2, 0, 0),
+	V2_0_3(2, 0, 3),
+	V2_2(2, 2, 0),
+	V3(3, 0, 0);
+	
+	private final int major;
+	private final int minor;
+	private final int patch;
+
+	Version(int major, int minor, int patch) {
+		this.major = major;
+		this.minor = minor;
+		this.patch = patch;
+	}
+
+	public boolean isLowerOrEqualTo(Version other) {
+		if (major < other.major) {
+			return true;
+		} else if (major == other.major) {
+			if (minor < other.minor) {
+				return true;
+			} else if (minor == other.minor) {
+				return patch <= other.patch;
+			}
+		}
+		return false;
+	}
```

Comment:
This could probably be made faster by comparing enum ordinals. But maybe you liked the version checks to be explicit.

Reply:
Good suggestion. We do need to make sure that the enum values are in the right order then, and that we keep them that way, or we need to add a simple value ourselves?